tests/mtouch/RegistrarTest.cs

```diff
@@ -1414,6 +1414,43 @@ class CTP4 : CTP3 {
 			}
 		}
 
+		[Test]
+		public void LinkedAwayGenericTypeAsOptionalMemberInProtocol ()
+		{
+			// Normally compilable test code can go into monotouch-test, but in this case it depends on the
+			// generic type (NSSet) being linked away, and since monotouch-test uses NSSet, it's not linked
+			// away, and we don't hit the required condition causing the bug.
```

Comment:
maybe `link sdk` or `link all` would have fit the bill ?

Reply:
Good idea, I've put it in `link all` instead.